test/Microsoft.AspNetCore.Routing.Tests/InlineRouteParameterParserTests.cs

```diff
@@ -29,7 +29,7 @@ public void ParseRouteParameter_WithoutADefaultValue(string parameterName)
         }
 
         [Fact]
-        public void ParseRouteParameter_WithoutADefaultValue()
+        public void ParseRouteParameter_WithEmptyDefaultValue()
```

Comment:
Yeah, seems so. It tries to call in this method with theory parameters which xunit doesn't like. Could you get an issue filed for this?